Update MFA content (#341)
PrivSec-dev/privsec.dev: content/posts/knowledge/Multi-factor Authentication.md

- There is no firmware updates. When a vulnerability is found on your current firmware version, your only recourse is to buy a new key with patched firmware.
- The internal storage, at least for the FIDO2 interface, is not encrypted in a meaningful manner. Since there is no way to force a PIN for all FIDO2 operations, it is clear that the encryption key, if it even exists at all, is not protected with a secret only known to the user. With a Yubikey, you will be relying solely on the secure element and potting to protect your FIDO2 secrets.

### Nitrokey

Nitrokeys, much like the Yubikey, do not encrypt its FIDO2 storage in a meaningful manner. Additionally, it is explicitly noted in their app that the HOTP/TOTP secret storage is also [not encrypted](https://docs.nitrokey.com/nitrokeys/features/totp/general).

Some NitroKeys do support password storage, however, it is tied to the OpenPGP interface. Resetting the OpenPGP interface will make the password database [inaccessible](https://docs.nitrokey.com/nitrokeys/pro/factory-reset). 

> Nitrokeys support firmware updates and receieve them quite frequently. However, the they does not have any potting to protect themselves against physical attacks. 

Reviewer: ```suggestion
Nitrokeys support firmware updates and receive them quite frequently. However, they do not have any potting to protect themselves against physical attacks. 
```

### OnlyKey

> The OnlyKey encrypts its entire internal storage against the user PIN and static secrets baked into the hardware. The PIN consists of digits from 1 to 6, has the maximum length of 10 digits, and has to be physically typed on the key. The Onlykey supports a duress PIN, which other security keys do not have. For further protection against physical attacks, the key is also potted in resin.

Reviewer: ```suggestion
The OnlyKey encrypts its entire internal storage against the user PIN and static secrets baked into the hardware. The PIN consists of digits from 1 to 6, has the maximum length of 10 digits, and has to be physically typed on the key. The OnlyKey supports a duress PIN, which other security keys do not have. For further protection against physical attacks, the key is also potted in resin.
```
